kazoo/recipe/lock.py

```diff
@@ -242,18 +253,34 @@ def _inner_acquire(self, blocking, timeout):
             finally:
                 self.client.remove_listener(self._watch_session)
 
-    def acquired_lock(self, children, index):
-        return index == 0
+    def predecessor(self, children, index):
+        for c in children[:index]:
+            if any(n in c for n in self._EXCLUDE_NAMES):
```

Comment:
Just wanted to check, is this logic right, it seems to be only finding children that have names in exclude names, should it be not in ?